src/drivers/mpu6000/mpu6000.cpp

```diff
@@ -1288,6 +1329,7 @@ MPU6000::ioctl(struct file *filp, int cmd, unsigned long arg)
 					float cutoff_freq_hz = _accel_filter_x.get_cutoff_freq();
 					float sample_rate = 1.0e6f / ticks;
 					_set_dlpf_filter(cutoff_freq_hz);
+					_set_icm_acc_dlpf_filter(cutoff_freq_hz);
```

Comment:
@JochiPochi 

My take on these things is to program defensively. Register 0x1d may not be documented in the MPU HW but it might be really be used "for what i don't known".

So as a general of thumb, I would wrap the all device specific code in a if (is_icm_device()) to be on the safe side.